pytorch/training/docker/1.9/py3/cu111/Dockerfile.gpu

```diff
@@ -0,0 +1,318 @@
+FROM nvidia/cuda:11.1.1-cudnn8-devel-ubuntu20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.8.10
+ARG CUBLAS_VERSION=11.3.0.106
+ARG OPEN_MPI_PATH=/opt/amazon/openmpi
+ARG EFA_PATH=/opt/amazon/efa
+
+ARG CUDA_HOME=/usr/local/cuda
+ARG CONDA_PREFIX=/opt/conda
+ARG METIS=metis-5.1.0
+ARG RMM_VERSION=0.15.0
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.9
```

Comment:
Can you update this to 1.0.10?

Reply:
@ndodda-amazon This is the PR that we are using for pt 1.9 release https://github.com/aws/deep-learning-containers/pull/1132. Do you mean to suggest updating SMDEBUG_VERSION there?

Reply:
My mistake, I thought this was the PR being used for the PT 1.9 release. 

Yes, could you please update the new Dockerfiles in #1132 to use smdebug 1.0.10?